chess/__init__.py

```diff
@@ -2465,6 +2465,48 @@ def __unicode__(self, reverse_color=False, borders=False):
             string += '   a b c d e f g h'
         return string
 
+    def __html__(self):
+        """
+        Returns a html-version of the board.
+        Can be displayed in e.g. an IPython notebook using
+        IPython.display.HTML(board.__html__())
+        """
+        tr = '<tr style="vertical-align:bottom;">'
+        string = '<table style="text-align:center; \
+                                border-spacing:0pt; \
+                                font-family:\'Arial Unicode MS\'; \
+                                border-collapse:collapse; \
+                                border-color: black; \
+                                border-style: solid; \
+                                border-width: 0pt 0pt 0pt 0pt">'    
+        for rank in range(8,0,-1):
+            string += tr
+            string += '<td style="vertical-align:middle; \
+                                  width:12pt">%d</td>' % rank
+            for i,file_ in enumerate('A B C D E F G H'.split()):
+                square = eval('%s%d' % (file_,rank))
```

Comment:
Oops, I fixed this before but it reverted somehow... I think I branched from before the fix.  I'll take care of it.